config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  resources :categories do
+    resources :purchases
+  end
+
+  devise_for :users
+  root 'categories#index'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+end
```

Comment:
- [ ] You did a great job on the project but you missed adding a splash page(Homa page) for the application as requested in the project requirements. Please make sure you add that page with links to sign-up and login pages and it should be accessed without authentication.

![image](https://user-images.githubusercontent.com/83803414/201387837-0befd0e8-5d10-48da-ba1a-4774601fde09.png)